packages/commons/src/assignment.ts

```diff
@@ -34,8 +35,10 @@ export const findAssignment = <T extends SavedBundle>(bundle: T) => {
   const latestStatus = getStatusFromTask(task)
 
   for (const task of allTasks) {
-    // if the first task found is unassignment, it's sure that the bundle is unassigned
-    const isUnassigned = findExtension(UNASSIGNED_EXTENSION_URL, task.extension)
+    // if the first task found is unassignment or makeCorrection, it's sure that the bundle is unassigned
+    const isUnassigned =
+      findExtension(UNASSIGNED_EXTENSION_URL, task.extension) ||
+      findExtension(MAKE_CORRECTION_EXTENSION_URL, task.extension)
```

Comment:
@jamil314 wouldn't the root cause of this issue be where the task with the `MAKE_CORRECTION` extension is created? I think that's where we should be adding an unassignment if we aren't at the moment.